gc/base/standard/ConcurrentGCSATB.cpp

```diff
@@ -451,6 +451,8 @@ MM_ConcurrentGCSATB::completeConcurrentTracing(MM_EnvironmentBase *env, uintptr_
 {
 	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
 
+	GC_OMRVMInterface::flushCachesForGC(env);
+
```

Comment:
just caught my eye: preserve/restoreGlobalFragmentIndex should have disable/enableWritterBarrier wrappers (although it's a dilemma if they should belong to specific class like MM_RememberedSetSATB or be more generic like a virtual method of GlobalCollector (as it already exists in RealtimeGC). Up to you if/how you want to change. Obviously, low priority.

Reply:
I took a quick look, much of the surrounding code should be considered to be moved into the wrapper as well, I've made a TODO note will return to this